src/templates/en/2021/methodology.html

```diff
@@ -214,7 +214,7 @@ <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a><
           <tr>
             <td>Connection</td>
             <td>Cable (5/1 Mbps 28ms RTT)</td>
-            <td>4G (9 Mbps 170ms RTT)</td>
+            <td>3G (1.600/0.768 Mbps 300ms RTT)</td>
```

Comment:
We [discovered that 2021 actually used 3G not 4G](https://github.com/HTTPArchive/httparchive.org/issues/605#issuecomment-1146080845) so updated this top same as 2020.